### PR TITLE
Fix for audio issie

### PR DESCRIPTION
**Pull request by faisalmohd83**

### CODE REVIEW

**Comment by faisalmohd83** [comment]
@Azeemsha 
@febincp6 
please review

File: podcast.rss [hunk 1]
```diff
@@ -1930,7 +1930,7 @@
       <description>
         <![CDATA[<p>Velicham Qur'an Dars series</p><p>Qur'an class #139</p><p>💠Surah At-Takathur💠</p><p>ആയത്ത് -1<br />വിവരണം -1</p><p>أَلْهَىٰكُمُ ٱلتَّكَاثُرُ <br /> പരസ്പരം പെരുമനടിക്കല്&zwj; നിങ്ങളെ അശ്രദ്ധരാക്കിയിരിക്കുന്നു.</p><p>🔸Admin Team Velicham Onlive🔸</p>]]>
       </description>
-      <enclosure url="https://velicham-onlive.github.io/audios/102-takathur/2-vqd-podcast-unit102.m4a" type="audio/m4a" length="854000"/>
+      <enclosure url="https://velicham-onlive.github.io/audios/102-takathur/2-vqd-podcast-unit102.mp3" type="audio/mp3" length="2196000"/>
       <itunes:duration>09:07</itunes:duration>
```
Comment (faisalmohd83) [inline podcast.rss:1934]: Is the audio duration same after mp3 conversion?

File: podcast.rss [hunk 2]
```diff
@@ -2185,7 +2185,7 @@
       <description>
         <![CDATA[<p>Velicham Qur'an Dars series</p><p>Qur'an class #162</p><p>💠Surah Al-Qaria💠</p><p>ആയത്ത് - 8</p><p>وَأَمَّا مَنْ خَفَّتْ مَوَٰزِينُهُۥ <br /> എന്നാൽ യാതൊരുവന്റെ&nbsp;നന്മയുടെ തുലാസുകൾ ഭാരം കുറഞ്ഞുവോ.</p><p>🔸Admin Team Velicham Onlive🔸</p>]]>
       </description>
-      <enclosure url="https://velicham-onlive.github.io/audios/101-qaria/8-vqd-podcast-unit101.m4a" type="audio/m4a" length="548470"/>
+      <enclosure url="https://velicham-onlive.github.io/audios/101-qaria/8-vqd-podcast-unit101.mp3" type="audio/mp3" length="1393000"/>
       <itunes:duration>05:51</itunes:duration>
```
Comment (faisalmohd83) [inline podcast.rss:2189]: Is the audio duration same after mp3 conversion?

File: podcast.rss [hunk 3]
```diff
@@ -2409,7 +2409,7 @@
       <description>
         <![CDATA[<p>Velicham Qur'an Dars Series</p><p>Qur'an class #182</p><p>💠Surah Al-Zalzala💠<br />ആയത്ത്-1&zwnj;</p><p>إِذَا زُلْزِلَتِ ٱلْأَرْضُ زِلْزَالَهَا <br /> ഭൂമി അതിന്റെ വിറപ്പിക്കൽ വിറപ്പിക്കപ്പെട്ടാൽ.</p><p>🔸Admin Team Velicham Onlive🔸</p>]]>
       </description>
-      <enclosure url="https://velicham-onlive.github.io/audios/99-zalzala/3-vqd-podcast-unit99.m4a" type="audio/m4a" length="703940"/>
+      <enclosure url="https://velicham-onlive.github.io/audios/99-zalzala/3-vqd-podcast-unit99.mp3" type="audio/mp3" length="1889000"/>
       <itunes:duration>07:30</itunes:duration>
```
Comment (faisalmohd83) [inline podcast.rss:2413]: Is the audio duration same after mp3 conversion?

File: podcast.rss [hunk 4]
```diff
@@ -2420,7 +2420,7 @@
       <description>
         <![CDATA[<p>Velicham Qur'an Dars Series</p><p>Qur'an class #183</p><p>💠Surah Al-Zalzala💠<br />ആയത്ത്-2</p><p>وَأَخْرَجَتِ ٱلْأَرْضُ أَثْقَالَهَا <br /> ഭൂമി അതിന്റെ ഭാരങ്ങള്&zwj; പുറംതള്ളുകയും ചെയ്താൽ.</p><p>🔸Admin Team Velicham Onlive🔸</p>]]>
       </description>
-      <enclosure url="https://velicham-onlive.github.io/audios/99-zalzala/4-vqd-podcast-unit99.m4a" type="audio/m4a" length="869570"/>
+      <enclosure url="https://velicham-onlive.github.io/audios/99-zalzala/4-vqd-podcast-unit99.mp3" type="audio/mp3" length="2260000"/>
       <itunes:duration>09:16</itunes:duration>
```
Comment (faisalmohd83) [inline podcast.rss:2424]: Is the audio duration same after mp3 conversion?

File: podcast.rss [hunk 5]
```diff
@@ -1572,8 +1572,8 @@
       <description>
         <![CDATA[<p>Velicham Qur'an Dars series</p><p>Qur'an class # 107</p><p>💠Surah Al-Feel 💠<br />ആയത്ത്-1</p><p>ِ(أَلَمْ تَرَ كَيْفَ فَعَلَ رَبُّكَ بِأَصْحَابِ الْفِيلِ<br />ആനക്കാരോട്&zwnj; നിന്റെ രക്ഷിതാവ്&zwnj; എങ്ങനെ പ്രവർത്തിച്ചുവെന്ന് നീ കണ്ടില്ലേ?</p><p>🔸Admin Team Velicham Onlive🔸</p>]]>
       </description>
-      <enclosure url="https://velicham-onlive.github.io/audios/105-feel/7-vqd-podcast-unit105.m4a" type="audio/m4a" length="893000"/>
-      <itunes:duration>08:40</itunes:duration>
+      <enclosure url="https://velicham-onlive.github.io/audios/105-feel/7-vqd-podcast-unit105.mp3" type="audio/mp3" length="4072000"/>
```
Comment (faisalmohd83) [inline podcast.rss:1575]: Incorrect name for new file, please correct

**Review by faisalmohd83 — COMMENTED** [review]
1. 105-feel folder contains new images, please remove all newly added ones.
2. 105-feel/**7-vqd-podcast-unit106.m4a** renamed to incorrectly as **unit105.mp3**
3. Other review comments are inline

File: podcast.rss (hunk 1 above)
Comment (rahmanthoufeeque) [reply podcast.rss:1934]: yes, the duration is same after conversion.

File: podcast.rss (hunk 2 above)
Comment (rahmanthoufeeque) [reply podcast.rss:2189]: yes

File: podcast.rss (hunk 3 above)
Comment (rahmanthoufeeque) [reply podcast.rss:2413]: yes

File: podcast.rss (hunk 5 above)
Comment (rahmanthoufeeque) [reply podcast.rss:1575]: the file name was changed according to the other files in the folder.(was previously named as 106.mp3 under unit 105).

Comment (faisalmohd83) [reply podcast.rss:1575]: Good catch

**Review by faisalmohd83 — COMMENTED** [review]
Looks good to me

**Pull request merged**